doc/data_ops.rst

```diff
@@ -0,0 +1,71 @@
+.. _user_guide_data_ops_index:
+
+Complex multi-table pipelines with Data Ops
+===========================================
+
+Skrub provides an easy way to build complex, flexible machine learning pipelines.
+There are several needs that are not easily addressed with standard scikit-learn
+tools such as :class:`~sklearn.pipeline.Pipeline` and
+:class:`~sklearn.compose.ColumnTransformer`, and for which the Skrub DataOps offer
+a solution:
+
+- Multiple tables: We often have several tables of different shapes (for
+  example, "Customers", "Orders", and "Products" tables) that need to be
+  processed and assembled into a design matrix ``X``. The target ``y`` may also
+  be the result of some data processing. Standard scikit-learn estimators do not
+  support this, as they expect right away a single design matrix ``X`` and a
+  target array ``y``, with one row per observation.
+- DataFrame wrangling: Performing typical DataFrame operations such as
+  projections, joins, and aggregations should be possible and allow leveraging
+  the powerful and familiar APIs of `Pandas <https://pandas.pydata.org>`_ or
+  `Polars <https://docs.pola.rs/>`_.
+- Hyperparameter tuning: Choices of estimators, hyperparameters, and even
+  the pipeline architecture can be guided by validation scores. Specifying
+  ranges of possible values outside of the pipeline itself (as in
+  :class:`~sklearn.model_selection.GridSearchCV`) is difficult in complex
+  pipelines.
+- Iterative development: Building a pipeline step by step while inspecting
+  intermediate results allows for a short feedback loop and early discovery of
+  errors.
+
+In this section we cover all about the skrub Data Ops, from starting out with a
+simple example, to more advanced concepts like parameter tuning and and pipeline
+validation.
```

Comment:
The below does not render great in the site's table of contents (because the titles on this page are not exposed on the site's table of content) but I don't know how to fix this, so I'm moving on for now.